applications/GeoMechanicsApplication/tests/test_thermal_element/test_thermal_fixed_temperature/README.md

```diff
@@ -0,0 +1,25 @@
+# Test Cases for Thermal fixed temperature
+
+**Author:** [Mohamed Nabi](https://github.com/mnabideltares)
+
+**Source files:** [Thermal fixed temperature](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_thermal_element/test_thermal_fixed_temperature)
+
+
+## Case Specification
+In this thermal test case, a 1 x 1 m soil is considered, with the left side set at 0 degrees and the right side set at 25 degrees. The simulation spans 100 days to allow for a transition from an exponential to a linear temperature profile between the two sides. This test is conducted for various configurations, including 2D3N, 2D4N, 2D6N, 2D8N, 2D9N, 2D10N, 2D15N, 3D4N, and 3D10N. The temperature distribution within the box is then evaluated with its own result.
```

Comment:
The initial profile is not an exponential one, it is a step function ( starts from 0 everywhere, then a sudden jump to 25 at the right boundary ).